lib/elixir/lib/macro/env.ex

```diff
@@ -96,7 +96,7 @@ defmodule Macro.Env do
         ]
 
   @type expand_import_opts :: [
-          allow_locals: boolean(),
+          allow_locals: boolean() | (Macro.metadata(), atom(), arity(), [atom()], t() -> any()),
```

Comment:
Do we return `any` or does it have to be AST?

Reply:
Actually the return type is `false | function()`

Reply:
Ah, so we should update it!